partner_multi_image/models/res_partner.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# © 2009 Sharoon Thomas Open Labs Business Solutions
+# © 2014 Serv. Tecnol. Avanzados Pedro M. Baeza
+# © 2015 Antiun Ingeniería S.L. - Jairo Llopis
+# © 2016 Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+"""Reference core image fields to multi-image variants."""
+
+
+from openerp import models, fields
+
+
+class ResPartner(models.Model):
+
+    _name = "res.partner"
+    _inherit = [_name, "base_multi_image.owner"]
+
+    image = fields.Binary(
+        related='image_main',
+        store=False,
+        multi=False
```

Comment:
It's weird that you don't get affected by https://github.com/odoo/odoo/issues/10799. These fields are found in core Odoo, or they are new? If found in core, and they have `multi` there, you'll probably have to declare them with v7 style to avoid failures.